tool/github.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright 2020 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Fast fail the script on failures.
+set -e
+
+export JAVA_HOME=$JAVA_HOME_11_X64
+
+git clone --depth 1 https://github.com/flutter/flutter.git ../flutter
+export PATH="$PATH":`pwd`/../flutter/bin:`pwd`/../flutter/bin/cache/dart-sdk/bin
+flutter config --no-analytics
+flutter doctor
+export FLUTTER_SDK=`pwd`/../flutter
+
+if [ "$IDEA_VERSION" = "4.0" -o "$IDEA_VERSION" = "4.1" ] ; then
```

Comment:
What does this if statement do? Perhaps add a comment line (as bash if statements can be mysterious for people).

Reply:
I updated the comment in the block.